test/Microsoft.DocAsCode.MarkdigEngine.Tests/HtmlMarkdownObjectValidatorProvider.cs

```diff
@@ -6,11 +6,10 @@ namespace Microsoft.DocAsCode.MarkdigEngine.Tests
     using System.Collections.Immutable;
     using System.Composition;
 
-    using MarkdigEngine.Extensions;
-
     using Markdig.Syntax;
     using Microsoft.DocAsCode.Common;
-
+    using Microsoft.DocAsCode.MarkdigEngine.Validators;
+    
     [Export(ContractName, typeof(IMarkdownObjectValidatorProvider))]
```

Comment:
Does the test require to have the Export attribute based on the new design we talked ?